evap/evaluation/templates/base.html

```diff
@@ -185,6 +202,40 @@
             };
         </script>
 
+        <script type="module">
+            // Notebook logic
+
+            import { getCookie, setCookie} from "{% static 'js/utils.js' %}";
+            if (getCookie('evap_notebook_open') == "true") {
+                new bootstrap.Collapse(document.querySelector('#notebook'));
+                new bootstrap.Collapse(document.querySelector('#notebookButton'));  
+                onShowNotebook();
+            };
+
+            // make evap go away for notebook
+            $('#notebook').on('show.bs.collapse', function () {
```

Comment:
Although there is still quite some amount of jquery code in the evap, please avoid adding more to it :)